lib/Resources/config/internal.yml

```diff
@@ -5,6 +5,14 @@ parameters:
     netgen.ezplatform_site.root_location_id: 2
 
 services:
+    #
+    netgen.ezplatform_site.aggregate_repository:
+        class: Netgen\EzPlatformSiteApi\Core\Repository\Aggregate\Repository
+        arguments:
```

Comment:
This should be a private service by default.

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/67/commits/a822caa4a2a7259446eb521e155a0c735f705f63.